kolibri/plugins/coach/assets/src/views/quizzes/CreateExamPage/sidePanels/QuizResourceSelection/index.vue

```diff
@@ -364,15 +365,31 @@
 
       const { annotateTopicsWithDescendantCounts } = useQuizResources();
 
-      const unusedQuestionsCount = content => {
-        const questionItems = content.assessmentmetadata.assessment_item_ids.map(
-          aid => `${content.id}:${aid}`,
-        );
-        const questionsItemsUnused = questionItems
-          .filter(questionItem => !allQuestionsInQuiz.value.some(q => q.item === questionItem))
-          .filter(questionItem => !workingQuestions.value.some(q => q.item === questionItem));
-        return questionsItemsUnused.length;
-      };
+      const unusedQuestionsCount = useMemoize(content => {
```

Comment:
using useMemoize here is pretty smart

Reply:
I stole it from our 0.17 implementation: https://github.com/learningequality/kolibri/blame/release-v0.17.x/kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ResourceSelection.vue#L492